app/views/entities/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render 'devise/shared/navbar' %>
+<div class="group-wrapper">
+  <% if @entities.empty? %>
+    <p class="no-trans">No transactions yet</p>
+  <% else %>
+    <% @entities.each do |entity|%>
+      <%= link_to(entity_path(entity)) do %>
+        <ul class="group-container">
+          <li>
+              <img class='img' src="<%= entity.group.icon %>" alt="icon">
+              <h3><%= entity.name %></h3>
+              <%= entity.created_at.to_date.to_s %>
+            <h4 class="group-amount">
+              <%= entity.amount %>
+            </h4>
+          </li>
+        </ul>
+      <% end %>
+    <% end %>
+  <% end %>
+  <div class="a-link cat-leave">
+    <%= link_to('Back to groups', groups_path, class: "leave grp-leave") %>
+    <%= link_to(new_entity_path, class: 'new-grp') do %>
+      <button type="button">Add Transaction</button>
+    <% end %>
+  </div>
```

Comment:
- [ ] The footer button should change based on the page, if you are in home page, you should see only 1 link (as provided in design) which is add category, if you are inside the category you should see add transaction button. But currently I see another red button

Your design  | Figma design
------------- | -------------
<img width="452" alt="Screenshot 2022-06-03 at 22 29 29" src="https://user-images.githubusercontent.com/77038610/171924723-e4f0efe0-cf39-4b90-89ba-90fc998cd9c1.png"> <img width="456" alt="Screenshot 2022-06-03 at 22 30 28" src="https://user-images.githubusercontent.com/77038610/171924858-ebbdf23c-5c4c-4e33-9f80-e9ee17b88b4d.png">| <img width="229" alt="Screenshot 2022-06-03 at 22 29 57" src="https://user-images.githubusercontent.com/77038610/171924799-0745f382-f0b0-4c9a-babf-091a776e231b.png"> <img width="211" alt="Screenshot 2022-06-03 at 22 31 02" src="https://user-images.githubusercontent.com/77038610/171924960-fe5b19a5-8ec0-4751-860a-dd60a7bc4a0d.png">


- [ ] Kindly move Go back button where necessary, to the navbar as provided in design
<img width="229" alt="Screenshot 2022-06-03 at 22 40 04" src="https://user-images.githubusercontent.com/77038610/171926341-71ea9976-ae5c-4678-be47-f6952812ba7f.png">